docs/reference/query-dsl/mlt-query.asciidoc

```diff
@@ -241,6 +241,11 @@ number of terms that must match.
 The syntax is the same as the <<query-dsl-minimum-should-match,minimum should match>>.
 (Defaults to `"30%"`).
 
+`fail_on_unsupported_term`::
+Controls whether the query should fail if it encounters a term that is not
```

Comment:
maybe this would be clearer to say that it will fail (throw an exception) if any field within the supplied fields is not of the supported types (text, keyword).